torch/_dynamo/backends/torchxla.py

```diff
@@ -12,20 +12,6 @@
 
 
 @register_experimental_backend
-def torchxla_trivial(gm, fake_tensor_inputs):
-    return gm
-
-
-@register_experimental_backend
-def torchxla_trace_once(model, fake_tensor_inputs):
-    warnings.warn(
-        "This backend will be deprecated in 2.2, please use `openxla` backend instead"
-    )
-
-    return xla_backend_helper(model, fake_tensor_inputs)
-
-
-@register_backend
 def openxla_eval(model, fake_tensor_inputs):
```

Comment:
So we downgrade it from `backend` to `experimental_backend`

Reply:
right, it can still be used, but won't show up with `torch._dynamo.list_backends()`